### DISCUSSION TITLE
"skopeo" dependency can be removed

### DISCUSSION THREAD

**Discussion by alivenets** [containers/podman#14073] [Ideas]
Does podman need a skopeo dependency? Seems that skopeo binary is not used directly, but only needs the configuration files storage.conf and policy.json which are provided with skopeo.

**Comment by mheon** [comment]
Podman does not have any direct dependency on the Skopeo binary.

What distro is packaging things in this way?

**Comment by alivenets** [comment]
This is stated [here](https://github.com/containers/podman/blob/32748492e919b9f3a7882dc2febc454d39c04c3e/podman.spec.rpkg) and in [Yocto recipe](https://git.yoctoproject.org/meta-virtualization/tree/recipes-containers/podman/podman_git.bb#n110). My assumption is that it is only due to default config files.

**Reply by rhatdan** [reply]
Skopeo at one time shipped configuration files in a subpackage.  We moved these files to containers-common package years ago.
Podman should NOT require skopeo at all (And Skopeo should not require Podman).  Both should require containers-common or a subpackage that provides files like storage.conf, policy.json, containers.conf ...

**Reply by Luap99** [reply]
https://github.com/containers/podman/blob/32748492e919b9f3a7882dc2febc454d39c04c3e/podman.spec.rpkg#L128

This line is for the podman-tests package. I guess some system test requires skopeo. Podman itself does not require skopeo.

**Reply by alivenets** [reply]
Thank you very much for the detailed answer. Seems one needs to inroduce containers-common recipe in meta-virtualization to decouple from skopeo.
